hbase_sql/src/main/java/com/splicemachine/derby/impl/SpliceSparkKryoRegistrator.java

```diff
@@ -872,5 +873,14 @@ public DecimalBufferedSumAggregator read(Kryo kryo,Input input,Class type){
         instance.register(Vector.class);
         instance.register(KafkaReadFunction.Message.class,EXTERNALIZABLE_SERIALIZER);
         instance.register(TriggerDescriptorV4.class,EXTERNALIZABLE_SERIALIZER);
+        instance.register(SPSDescriptor.class,EXTERNALIZABLE_SERIALIZER);
+        try {
+            instance.register(Class.forName("splice.com.google.common.collect.EmptyImmutableList"));
```

Comment:
Why are you using Class.forName here?

Reply:
EmptyImmutableList is package private and not directly accessible.